src/app/pages/ShipyardPage.jsx

```diff
@@ -340,7 +355,7 @@ export default class ShipyardPage extends Page {
             </tbody>
           </table>
           <div style={{ overflowX: 'scroll', maxWidth: '100%' }}>
-            <table style={{ marginLeft: 'calc(12em - 1px)', zIndex: 0 }}>
+            <table style={{ marginLeft: 'calc(12em - 1px)', zIndex: 0 }} className="shipyard-table">
```

Comment:
Another double quotes.

Reply:
HTML attributes are consistantly double quotes

Reply:
Alright, just spotted it here and wasn't sure.